Samples.Objc/Samples.ObjC/API/Requests/CreateOrderRequest.m

```diff
@@ -10,35 +10,127 @@
 
 @implementation CreateOrderRequest
 
-- (id)initWith:(OrderIntent)intent purchaseUnits:(NSArray *)purchaseUnits {
+- (id)initWithOrder:(PPCOrderRequest*)order accessToken:(NSString*)token {
   self = [super init];
   if (self) {
-    self.intent = intent;
-    self.purchaseUnits = purchaseUnits;
+    self.order = order;
+    self.accessToken = token;
   }
   return self;
 }
 
 - (NSDictionary *)properties {
+  NSString *intentString = @"";
+  if (self.order.intent == PPCOrderIntentCapture) {
+    intentString = @"CAPTURE";
+  } else {
+    intentString = @"AUTHORIZE";
+  }
```

Comment:
Instead of giving this a default value when you declare it, you can just define that the variable exists, ie 
```objc
  NSString * intentString;
  if (self.order.intent == PPCOrderIntentCapture) {
    intentString = @"CAPTURE";
  } else {
    intentString = @"AUTHORIZE";
  }
```